docs/guides/dataloaders.md

```diff
@@ -118,6 +118,62 @@ the list for each incorrect key. A call with `keys == [1, 3]` returns
 directly. If the `load_users` function raises an exception, even `load`s with an
 otherwise valid key, like `await loader.load(1)`, will raise that exception.
 
+### Cache invalidation
+
+By default DataLoaders use an internal cache. It is great for performance, however it can cause problems when the data is modified
+(i.e., a mutation), as the cached data is no longer be valid! 😮
+
+To fix it, you can explicitly invalidate the data in the cache, using one of these ways:
+
+- Specifying a key with `loader.invalidate(id)`,
+- Specifying several keys with `loader.invalidate_many([id1, id2, id3, ...])`,
+- Invalidating the whole cache with `loader.invalidate_all()`
```

Comment:
should we add a note that this is the same a clear in dataloader js?

and same for preset being prime in dataloader js?

Reply:
Good point!

I really didn't think about keeping consistency between this and the JS api, but it sounds like a good idea

Reply:
I renamed the method to `clear_*` and `prime_*`